sources/osg/CullFace.js

```diff
@@ -22,28 +19,32 @@ CullFace.FRONT_AND_BACK = 0x0408;
 
 /** @lends CullFace.prototype */
 CullFace.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( StateAttribute.prototype, {
+
     attributeType: 'CullFace',
+
     cloneType: function () {
         return new CullFace();
     },
+
     setMode: function ( mode ) {
-        if ( typeof mode === 'string' ) {
-            mode = CullFace[ mode ];
-        }
-        this._mode = mode;
+        var value = mode;
+        if ( typeof value === 'string' ) value = CullFace[ value ];
+        this._mode = value;
     },
+
     getMode: function () {
         return this._mode;
     },
+
     apply: function ( state ) {
         var gl = state.getGraphicContext();
-        if ( this._mode === CullFace.DISABLE ) {
+        if ( !this._mode ) {
```

Comment:
Myself not very fan of undocumenting code for a very small gain.
( as in Enum Use should be value independant. )